frontend/src/views/groupmember/UpdateNumberAndDept.vue

```diff
@@ -173,3 +182,13 @@ export default {
   },
 }
 </script>
+
+<style scoped>
+p {
+  font-weight: bold;
+  color: #003366;
+}
+h2 {
+  color: #003366;
+}
+</style>
```

Comment:
I don't think this styling is required. Despite the mockup, blue isn't used for page text.